src/index.js

```diff
@@ -19,13 +48,15 @@ const getApplicationId = (token: string): string | null => {
     return null;
   }
 };
-
-const appDir: string = path.join(__dirname, "../app");
-const appPackage: Package = require(path.join(appDir, "package.json"));
-const { name, version }: Package = require(path.join(
-  __dirname,
-  "../package.json"
-));
+/**
+ * @type {string}
+ */
+const appDir = path.join(__dirname, "../app");
+/**
+ * @type {Package}
+ */
+const appPackage = require(path.join(appDir, "package.json"));
+const { name, version } = require(path.join(__dirname, "../package.json"));
```

Comment:
typing seems to work fine with destructure assignment, we can add the `Package` type hint here as well